### DISCUSSION TITLE
Fluent v9 component roadmap and upgrade guide

### DISCUSSION THREAD

**Discussion by sjwilczynski** [microsoft/fluentui#23829] [General] [answered]
Hey folks, our feature crew is considering using fluentui/react-components for the redesign of one of M365 components. We really love all the benefits in performance, styling engine and bundle size but we currently use some of the components that are still not scheduled in [the roadmap](https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap). Could someone share more details on when we could expect Dialog, Shimmer and ProgressIndicator to be part of the new package? 

We also found an [upgrade guide](https://psychic-waddle-c9811b2e.pages.github.io/upgrade-v8-v9/UpgradeGuide.html) but it contains a lot of TODOs. Is someone still actively working on it?

**Comment by spmonahan** [comment, marked as answer]
Hi @sjwilczynski,

The current upgrade guide is available on [react.fluentui.dev](https://react.fluentui.dev/?path=/docs/concepts-upgrading-overview--page) and is actively maintained and updated.

@paulgildea Can you share more details on the component road map?

**Reply by sjwilczynski** [reply]
Cool, thank you for pointing me to the most up to date resource. Will more shims components be added there? Right now, it's only `Stack` and having an easy way to create v8 theme from v9 and vice versa would be helpful in doing the migration in multiple steps :)

**Reply by spmonahan** [reply]
@GeoffCoxMSFT has been leading the upgrade effort and should be able to answer these questions for you.

**Reply by paulgildea** [reply]
@sjwilczynski As we're moving into FY23 planning, we're going to start to capture the roadmap in quarter-based increments. I don't have any specifics to share at this moment, but we will soon.

**Reply by sjwilczynski** [reply]
Looking forward to getting more details on the roadmap!
